ReflectorIOS/Helpers/Theme.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Theme.swift
+//  ReflectorIOS
+//
+//  Created by Nikhil Menon on 9/14/20.
+//  Copyright © 2020 Nikhil Menon. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+
+/// Contains custom colors  as `UIColor` used in the app.
+enum Theme {
+    static let reflectorMaroon = UIColor(named: "reflector-maroon")
```

Comment:
Force unwrap this buy putting a `!` at the end of `UIColor(named: "reflector-maroon")`

In this case, it's safe to unsafely unwrap this UIColor because we know for a fact that there is a color that has the string "reflector-maroon."